docs/source/tutorials/image.ipynb

```diff
@@ -30,7 +25,7 @@
     "Quickstart\n",
     "<br/>\n",
     "    \n",
-    "Already have a `model`? Run cross-validation to get out-of-sample `pred_probs` and then the code below to find any potential label errors in your dataset.\n",
+    "Already have a `model`? Run cross-validation to get out-of-sample `pred_probs` and `features` and then use the code below to find any potential issues in your dataset.\n",
```

Comment:
```suggestion
    "Already have a ML model? Run cross-validation to get out-of-sample `pred_probs` and provide `features` (embeddings of the data). Then use the code below to find any potential issues in your dataset (you can also run this code with one of `pred_probs` or `features` instead of both, but less issue types will be considered).\n",
```